packages/route-viewer-overlay/src/index.tsx

```diff
@@ -76,7 +76,7 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
 
         current?.fitBounds(bounds, {
           duration: 500,
-          padding: 200
+          padding: window.innerWidth > 500 ? 200 : undefined
```

Comment:
Is there some clever way we should think about doing this using a formula? Maybe a percentage of screen width? I'm also fine with this.

Reply:
Yeah we could try to somehow set the padding dynamically based on how much free room there is. For now though even this is still causing some problems in a few cases. 

Once we've resolved all the mobile view weirdnesses we should definitely do this!